htdocs/sass/rcloud-discover.scss

```diff
@@ -99,6 +99,10 @@ progress::-webkit-progress-value {
 .grid-item h1{
     font-size: 17px;
     margin: 0;
+    /* firefox */
+    white-space: pre-wrap;
+    word-break: break-all;
+    /* chrome */
     word-break: break-word;
```

Comment:
Yeah, it's [not on MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break), so I'm guessing it's non-standard. Chrome seems to be happy with `break-all`, could you verify and update your PR?